cumulusci/tasks/metadeploy.py

```diff
@@ -268,6 +275,8 @@ def _convert_org_providers_to_plan_equivalent(
             org_providers = "Both"
         elif providers == ["devhub"]:
             org_providers = "Scratch"
+        else:  # pragma: no cover
```

Comment:
This ensures that a None can never escape this function. According to the type signature, it never should have.

But @TheBitShepherd should verify that all cases were supposed to be covered here.